packages/grid/x-data-grid-generator/package.json

```diff
@@ -32,7 +32,7 @@
   "dependencies": {
```

Comment:
I think we should release the generator package to prevent this issue https://github.com/mui/mui-x/issues/9807#issuecomment-1653928455

Reply:
Interesting, so we must also release this whenever we release `x-data-grid-premium`, thanks for the update!

Reply:
Updated!

Reply:
> Is it worth documenting somewhere for upcoming release engineers

Did you use the `release:version` script to generate the version changes or have you done those manually?
As far as I can remember, whenever I used the script, the generator package version has always been bumped 🤔

Reply:
> Did you use the release:version script to generate the version changes

Yes, but it also bumped `x-license-pro` which wasn't something we had to to bump.

Reply:
Well, `x-license-pro` has been bumped by lerna, because it detected a change in the repo, but it simply was a bump of `MUI Core` dependencies. 🙈